### ISSUE TITLE
Completely leaving out the version number in Accept header versioning schemes with `strict: true` does not result in a 406

### ISSUE THREAD

**Issue by suan** [ruby-grape/grape#1206]
When having an endpoint with:

```
version 'v1', vendor: 'foo.resource', using: header, cascade: false, strict: true
content_type :v1_resource, 'application/vnd.foo.resource-v1+json'
format :v1_resource
```

Any request with both "Accept: application/json" and "Accept: application/vnd.foo.resource-v1+json" headers present does not result in a 406, whereas the expected behavior is that it _would_ be a 406, because of the `strict` option.

(When only the "Accept: application/vnd.foo.resource-v1+json" header is present it correctly responds with a 406)

**Comment by dblock** [comment]
That would be a problem. Could you please try this on HEAD? Some related issues may have been fixed.

If you have a moment, try to replicate this in a Grape test?

**Comment by dblock** [comment]
And I mean specifically https://github.com/ruby-grape/grape/pull/1101.

**Comment by stjhimy** [comment]
This can be closed since #1222 confirmed it was fixed.

**Comment by dblock** [comment]
Closing.
